src/frontend/src/layouts/PageBlock.tsx

```diff
@@ -10,6 +10,7 @@ import { useTheme } from '../hooks/Theme.hooks';
 interface PageBlockProps {
   title: string;
   headerRight?: ReactNode;
+  cardContainerStyle?: string;
```

Comment:
do we need this or can we wrap it in a div and put the styling on the div?

Reply:
this was added as part of ari's PR -- personally i like it because i think it makes it very clear what you're styling but im not a frontend expert so idk if thats good practice or not